test/strong_mode_test.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library analyzer_cli.test.strong_mode;
+
+import 'dart:io';
+import 'dart:mirrors';
+import 'package:path/path.dart' as path;
+import 'package:unittest/unittest.dart';
+
+/// End-to-end test for --strong checking.
```

Comment:
I didn't see any existing end-to-end tests for Driver/AnalysisImpl, so added this as a simple spawn process. Maybe I missed it though. I get nervous if my code isn't covered by at least something, so wanted to add something to test --strong mode and verify it prints extra errors :)